src/main/java/SearchPage.java

```diff
@@ -0,0 +1,19 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class SearchPage extends BasePage {
+
+    @FindBy(xpath = ".//div[@class='products fluid-grid clearer-block']")
```

Comment:
Can we use `contains()` for classes in the XPath?

Reply:
Done